README.md

```diff
@@ -33,7 +32,43 @@ See a demo of tobs in action by clicking the video below:
 
 # 🔥 Quick start
 
-## Installing the CLI tool
+## Prerequisites
+
+Using tobs to install full observability stack with openTelemetry support currently requires installation of cert-manager. To do install it please follow [cert-manager documentation](https://cert-manager.io/docs/installation/).
+
+*Note*: cert-manager is not required when using tobs with opentelemetry support disabled.
+
+## Installing the helm chart
+
+The following command will install Kube-Prometheus, OpenTelemetry Operator, TimescaleDB, and Promscale
+into your Kubernetes cluster:
+
+```
+helm repo add timescale https://charts.timescale.com/
+helm repo update
+helm install --wait <release_name> timescale/tobs
+```
+
+*Note*: `--wait` flag is necessary for successfull installation as tobs helm chart can create opentelemetry Custom Resources only after opentelemetry-operator is up and running. This flag can be omited when using tobs without opentelemetry support.
+
+For detailed configuration and usage instructions, take a look at the [helm chart's README](/chart/README.md).
+
+# Configuring the stack
+
+All configuration for all components happens through the helm values.yaml file.
+You can view the self-documenting [default values.yaml](chart/values.yaml) in the repo.
+We also have additional documentation about individual configuration settings in our
+[Helm chart docs](chart/README.md#configuring-helm-chart).
+
+# 🛠Alternative deployment methods
+
+## [DEPRECATED] Using the `tobs` CLI tool
+
+We also provide a CLI tool to deploy tobs on a Kubernetes cluster. 
+
+*NOTE*: At this point, the CLI tool is just a thin wrapper around the helm chart, and hence it's set to be removed in future releases. 
+
+### Installing the CLI tool
```

Comment:
Could we drop all sub-sections of cli installation and instead point to `cli/README.md` doc?